friends/src/components/Friend.js

```diff
@@ -0,0 +1,13 @@
+import React from "react"
+
+const Friend = props => {
+	return (
+		<div className="friend-card">
+			<h2>{props.friend.name}</h2>
```

Comment:
I would look into destructing props.friend so that you could make the code easier to read and less to type. something like `const {name, friend, age} = props.friend` Then using those three respective variables.